web_action_conditionable/static/src/js/views.js

```diff
@@ -0,0 +1,23 @@
+/*global openerp, _, $ */
+
+openerp.web_action_conditionable = function (instance) {
+  instance.web.View.include({
+    is_action_enabled: function(action) {
+      var attrs = this.fields_view.arch.attrs;
+      if (action in attrs) {
+	try {
+	  data = JSON.parse(attrs[action]);
+	  return data;
+	} catch(error) {
```

Comment:
does it make sense to inspect error and only do this if we had a json parse error?